### ISSUE TITLE
Installation of Git-Duet On M1 Macs

### ISSUE THREAD

**Issue by mjha4-ford** [git-duet/git-duet#97]
Facing difficulty with the installation of git-duet on m1 Mac even after using Rosetta. Also after specifying the architecture (arch -x86_64) it is not possible.

**Comment by jszwedko** [comment]
Hey @mjha4-ford! Thanks for the report. I'm guessing we probably just need to rebuild with a newer Go version.

**Comment by Haegi** [comment]
@jszwedko are you planning to release a new version of git-duet as well? The last release is already two years old and it would be nice to have the latest state of git-duet available via homebrew.

**Comment by jszwedko** [comment]
Apologies @Haegi , I neglected to follow through on releasing after upgrading the version of Go being used. I see @rafecolton has picked up #99 though, so I expect he'll release a new version soon.

**Comment by Haegi** [comment]
@jszwedko sounds nice, thank you very much.

**Comment by textbook** [comment]
As of v0.8.0, `git-duet` still cannot be installed (at least via Homebrew) on M1 machines:

```sh
$ brew install git-duet
git-duet: The x86_64 architecture is required for this software.
Error: git-duet: An unsatisfied requirement failed this build.
```

I think part of the fix for this would be adding `darwin/arm64` (_"macOS with Apple Silicon CPU"_ per mitchellh/gox#150) to the list of architectures in `scripts/dist`:

https://github.com/git-duet/git-duet/blob/e562d14145714efdf71ad76b748d822359596d35/scripts/dist#L9

**Comment by jszwedko** [comment]
Ah, yes, apologies, I misread the original issue as being related to OSX upgrades, which have required Go version upgrades before. I see this was asking about the M1 Macs though. I've opened #108 to add a release build for it.

**Comment by jszwedko** [comment]
I realized I was confusing this issue with https://github.com/git-duet/git-duet/issues/99

I've published a new release, 0.9.0, with support for `darwin/arm64` though. Let me know if that doesn't work for you!
